dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/descriptors/KeyJsonValueSchemaDescriptor.java

```diff
@@ -46,6 +46,7 @@ public Schema getSchema() {
     Schema schema = new Schema(DatastoreEntry.class, SINGULAR, PLURAL);
     schema.setRelativeApiEndpoint(API_ENDPOINT);
     schema.setOrder(9060);
+    schema.setDataShareable(true);
```

Comment:
This change was required, otherwise the sharing access did not take affect. Please confirm that this looks ok for anyone with knowledge in this area.